## 22% of end-end tests have passed
```
Total executed tests: 40
Failed tests: 31
Expected to fail tests: 0
Passed tests: 9
Not executed tests: 11
```
```
IDs of not executed tests: 702730,702782,702731,702745,702813,702855,702733,702783,702784,703391,703496 
```
```
IDs of failed tests: 727230,702809,703194,702859,702777,702732,702957,702851,702840,703495,703086,702948,727231,702894,703297,704615,703503,727229,703202,702839,702947,702838,702844,702843,702807,702808,727232,702845,702958,703629,702841 
```
<h3>Not executed tests (11)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/communities-and-group-members-name&PR_ID=20619&APK_NAME=20619.apk&TR_CASE_IDS=702730,702782,702731,702745,702813,702855,702733,702783,702784,703391,703496">Rerun not executed tests</a></li></details><h3>Failed tests (31)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/communities-and-group-members-name&PR_ID=20619&APK_NAME=20619.apk&TR_CASE_IDS=727230,702809,703194,702859,702777,702732,702957,702851,702840,703495,703086,702948,727231,702894,703297,704615,703503,727229,703202,702839,702947,702838,702844,702843,702807,702808,727232,702845,702958,703629,702841">Rerun failed tests</a></li><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602675">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div># STEP: Device 1 creates a new user</div><div># STEP: Device 2 sign in, user name is receiver</div></blockquote></p>

```
activity_center/test_activity_center.py:176: in test_add_contact_field_validation
    self.loop.run_until_complete(run_in_parallel(((_device_1_creates_user, {}),
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
activity_center/test_activity_center.py:166: in _device_1_creates_user
    self.profile_1.driver.reset()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/extensions/applications.py:299: in reset
    self.execute(Command.RESET)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/home/chef/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 install -r /tmp/tmpeNjJLo/StatusIm-Mobile-240709-073912-9bd5c2-pr20619-x86_64.apk' exited with code 1'; Command output: adb: device 'emulator-5554' not found
E   
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/home/chef/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 install -r /tmp/tmpeNjJLo/StatusIm-Mobile-240709-073912-9bd5c2-pr20619-x86_64.apk' exited with code 1'; Command output: adb: device 'emulator-5554' not found
E   
E       at getResponseForW3CError (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/errors.js:1073:9)
E       at asyncHandler (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/protocol.js:491:57)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9aaabac110a47ba807c101aef778653?auth=c6f3b36320dcbb1ff2cbe0407657c76c#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8db945ed57b74b42a7cb857128fb1083?auth=f001e87cf006b227f3383a8f5985fdf3#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602693">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><p><blockquote><div># STEP: Creating a new user on Device2</div></blockquote></p>

```
activity_center/test_activity_center.py:100: in test_activity_center_contact_request_accept_swipe_mark_all_as_read
    self.home_2.reopen_app(sign_in=False)
../views/base_view.py:706: in reopen_app
    app_package = self.driver.current_package
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/extensions/android/common.py:74: in current_package
    return self.assert_extension_exists(ext_name).execute_script(ext_name)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: Could not retrieve the currently focused package and activity. Original error: Error executing adbExec. Original error: 'Command '/home/chef/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell dumpsys window displays' exited with code 1'; Command output: adb: device 'emulator-5554' not found
E   
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not retrieve the currently focused package and activity. Original error: Error executing adbExec. Original error: 'Command '/home/chef/android-sdk-linux/platform-tools/adb -P 5037 -s emulator-5554 shell dumpsys window displays' exited with code 1'; Command output: adb: device 'emulator-5554' not found
E   
E       at getResponseForW3CError (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/errors.js:1073:9)
E       at asyncHandler (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/protocol.js:491:57)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9aaabac110a47ba807c101aef778653?auth=c6f3b36320dcbb1ff2cbe0407657c76c#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8db945ed57b74b42a7cb857128fb1083?auth=f001e87cf006b227f3383a8f5985fdf3#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602716">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td>

```
Test setup failed: critical/test_wallet.py:22: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602715">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td>

```
Test setup failed: critical/test_wallet.py:22: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602711">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Getting username card by index 1 </div><div>Device 1: Find `UserProfileElement` by `xpath`: `(//*[@content-desc='profile-card'])[1]` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:246: in test_community_discovery
    self.sign_in.get_user_profile_by_index(index=1).click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:84: in find_element
    raise exception
../views/base_element.py:77: in find_element
    return self.driver.find_element(self.by, self.locator)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/526bea1dd4c14bc4a61c29286c298f67?auth=fd411d539323b704b27899fd89c1e18b#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602670">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The test session has already finished, and can't receive further commands. For help please check https://docs.saucelabs.com/dev/error-messages
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602709">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The test session has already finished, and can't receive further commands. For help please check https://docs.saucelabs.com/dev/error-messages
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602706">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The test session has already finished, and can't receive further commands. For help please check https://docs.saucelabs.com/dev/error-messages
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602705">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The test session has already finished, and can't receive further commands. For help please check https://docs.saucelabs.com/dev/error-messages
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602680">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'Hey, admin!')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_group_chat.py:95: in test_group_chat_join_send_text_messages_push
    self.chats[1].chat_element_by_text(message_to_admin).wait_for_status_to_be('Delivered', timeout=120)
../views/chat_view.py:225: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/45e6a92699f1422badad0ef29d2ea886?auth=1ca6dd0c6ebe0af628ade15e65e1bb8d#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/874a4bca689d4858804efcfac606c810?auth=6fb1114ebdf111284ef4669718bc392e#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8003a5c73671470fb2bd5e2df247370b?auth=4685ef415b33012b2f252599b86a81c6#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602681">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The test session has already finished, and can't receive further commands. For help please check https://docs.saucelabs.com/dev/error-messages
```

<br/><br/></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602717">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td>

```
Test setup failed: critical/test_wallet.py:182: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(1)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602718">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td>

```
Test setup failed: critical/test_wallet.py:182: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(1)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602698">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:245: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602704">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602695">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602686">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602702">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602697">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602714">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602685">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602684">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bcee0ac3d34d496e969adf40168b6f42?auth=03d1dfb3450d1aacef1163f696ae13f2">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/85212a9310774e6fa75b040d83a7b812?auth=bb196c9f3681f434190f5a715b791587">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602689">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602688">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602687">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:321: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602682">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for chat: 'general' </div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:972: in test_community_markdown_support
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f970259f92ad42b08697fe7f1b10251f?auth=0f2e13ea02a67c4b843a8c6691cd3ff7#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b98c9cb2099b4f33bb0bb34674765d58?auth=dfc6e3f2d490f01346b72579e28edaa5#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602699">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:1027: in test_community_hashtag_links_to_community_channels
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f970259f92ad42b08697fe7f1b10251f?auth=0f2e13ea02a67c4b843a8c6691cd3ff7#207">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b98c9cb2099b4f33bb0bb34674765d58?auth=dfc6e3f2d490f01346b72579e28edaa5#173">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602690">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:1013: in test_community_leave
    self.home_2.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f970259f92ad42b08697fe7f1b10251f?auth=0f2e13ea02a67c4b843a8c6691cd3ff7#205">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b98c9cb2099b4f33bb0bb34674765d58?auth=dfc6e3f2d490f01346b72579e28edaa5#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602712">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:1127: in test_community_join_when_node_owner_offline
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f970259f92ad42b08697fe7f1b10251f?auth=0f2e13ea02a67c4b843a8c6691cd3ff7#210">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b98c9cb2099b4f33bb0bb34674765d58?auth=dfc6e3f2d490f01346b72579e28edaa5#175">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602700">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7e4544bd4adf42d3adb5c4570a434d2a?auth=da22b3e4021aee864b5878e45af8d7cf">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0bfc4c0e73cb4d7a9b3e24f11bc7c059?auth=734b6753b9914953dc4cd3283028951d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602701">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (9)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602713">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/74c2418b54f947b5aca6c7066ca01e95?auth=5849c700d0da5de62ccebbd8eca9ce04#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602674">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/74c2418b54f947b5aca6c7066ca01e95?auth=5849c700d0da5de62ccebbd8eca9ce04#219">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602679">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f970259f92ad42b08697fe7f1b10251f?auth=0f2e13ea02a67c4b843a8c6691cd3ff7#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b98c9cb2099b4f33bb0bb34674765d58?auth=dfc6e3f2d490f01346b72579e28edaa5#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602703">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/526bea1dd4c14bc4a61c29286c298f67?auth=fd411d539323b704b27899fd89c1e18b#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602672">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/526bea1dd4c14bc4a61c29286c298f67?auth=fd411d539323b704b27899fd89c1e18b#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602696">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/526bea1dd4c14bc4a61c29286c298f67?auth=fd411d539323b704b27899fd89c1e18b#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602691">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/526bea1dd4c14bc4a61c29286c298f67?auth=fd411d539323b704b27899fd89c1e18b#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602707">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/526bea1dd4c14bc4a61c29286c298f67?auth=fd411d539323b704b27899fd89c1e18b#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3602692">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f9aaabac110a47ba807c101aef778653?auth=c6f3b36320dcbb1ff2cbe0407657c76c#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8db945ed57b74b42a7cb857128fb1083?auth=f001e87cf006b227f3383a8f5985fdf3#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/modules/react-native-status/android/src/main/java/im/status/ethereum/module/EncryptionUtils.java
+++ b/modules/react-native-status/android/src/main/java/im/status/ethereum/module/EncryptionUtils.java
@@ -92,6 +92,11 @@ public class EncryptionUtils extends ReactContextBaseJavaModule {
         return Statusgo.hexToUtf8(str);
     }
 
+    @ReactMethod(isBlockingSynchronousMethod = true)
+    public String serializeLegacyKey(final String publicKey) {
+	return Statusgo.serializeLegacyKey(publicKey);
+    }
+
     @ReactMethod
     public void setBlankPreviewFlag(final Boolean blankPreview) {
         final SharedPreferences sharedPrefs = PreferenceManager.getDefaultSharedPreferences(this.reactContext);

--- a/modules/react-native-status/ios/RCTStatus/EncryptionUtils.m
+++ b/modules/react-native-status/ios/RCTStatus/EncryptionUtils.m
@@ -94,6 +94,10 @@ RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(hexToUtf8:(NSString *)str) {
     return StatusgoHexToUtf8(str);
 }
 
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(serializeLegacyKey:(NSString *)str) {
+    return StatusgoSerializeLegacyKey(str);
+}
+
 RCT_EXPORT_METHOD(setBlankPreviewFlag:(BOOL *)newValue)
 {
     NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];

--- a/modules/react-native-status/nodejs/status.cpp
+++ b/modules/react-native-status/nodejs/status.cpp
@@ -672,6 +672,38 @@ void _Sha3(const FunctionCallbackInfo<Value>& args) {
 
 }
 
+void _SerializeLegacyKey(const FunctionCallbackInfo<Value>& args) {
+	Isolate* isolate = args.GetIsolate();
+        Local<Context> context = isolate->GetCurrentContext();
+
+	if (args.Length() != 1) {
+		// Throw an Error that is passed back to JavaScript
+		isolate->ThrowException(Exception::TypeError(
+			String::NewFromUtf8Literal(isolate, "Wrong number of arguments for SerializeLegacyKey")));
+		return;
+	}
+
+	// Check the argument types
+
+	if (!args[0]->IsString()) {
+		isolate->ThrowException(Exception::TypeError(
+			String::NewFromUtf8Literal(isolate, "Wrong argument type for 'str'")));
+		return;
+	}
+
+
+	String::Utf8Value arg0Obj(isolate, args[0]->ToString(context).ToLocalChecked());
+	char *arg0 = *arg0Obj;
+
+	// Call exported Go function, which returns a C string
+	char *c = SerializeLegacyKey(arg0);
+
+	Local<String> ret = String::NewFromUtf8(isolate, c).ToLocalChecked();
+	args.GetReturnValue().Set(ret);
+	delete c;
+
+}
+
 void _ToChecksumAddress(const FunctionCallbackInfo<Value>& args) {
 	Isolate* isolate = args.GetIsolate();
         Local<Context> context = isolate->GetCurrentContext();
@@ -1976,6 +2008,7 @@ void init(Local<Object> exports) {
 	NODE_SET_METHOD(exports, "checkAddressChecksum", _CheckAddressChecksum);
 	NODE_SET_METHOD(exports, "isAddress", _IsAddress);
 	NODE_SET_METHOD(exports, "sha3", _Sha3);
+	NODE_SET_METHOD(exports, "serializeLegacyKey", _SerializeLegacyKey);
 	NODE_SET_METHOD(exports, "toChecksumAddress", _ToChecksumAddress);
 	NODE_SET_METHOD(exports, "logout", _Logout);
 	NODE_SET_METHOD(exports, "hashMessage", _HashMessage);

--- a/src/native_module/core.cljs
+++ b/src/native_module/core.cljs
@@ -333,6 +333,11 @@
              :key input-key})
   (.deserializeAndCompressKey ^js (encryption) input-key callback))
 
+(defn serialize-legacy-key
+  "Compresses an old format public key (0x04...) to the new one zQ..."
+  [public-key]
+  (.serializeLegacyKey ^js (encryption) public-key))
+
 (defn compressed-key->public-key
   "Provides compressed key to status-go and gets back the uncompressed public key via deserialization"
   [public-key deserialization-key callback]

--- a/src/status_im/subs/chats.cljs
+++ b/src/status_im/subs/chats.cljs
@@ -5,7 +5,8 @@
     [re-frame.core :as re-frame]
     [status-im.constants :as constants]
     [status-im.contexts.chat.events :as chat.events]
-    [status-im.contexts.profile.utils :as profile.utils]))
+    [status-im.contexts.profile.utils :as profile.utils]
+    [status-im.subs.contact.utils :as contact.utils]))
 
 (def memo-chats-stack-items (atom nil))
 
@@ -259,8 +260,19 @@
  :chats/photo-path
  :<- [:contacts/contacts]
  :<- [:profile/profile-with-image]
- (fn [[contacts {:keys [public-key] :as multiaccount}] [_ id]]
-   (let [contact (or (when (= id public-key) multiaccount) (get contacts id))]
+ :<- [:mediaserver/port]
+ :<- [:initials-avatar-font-file]
+ :<- [:theme]
+ (fn [[contacts {:keys [public-key] :as multiaccount} port font-file theme] [_ id]]
+   (let [contact (or (when (= id public-key) multiaccount)
+                     (get contacts id)
+                     (contact.utils/replace-contact-image-uri
+                      {:contact    {:public-key          id
+                                    :customization-color constants/profile-default-color}
+                       :port       port
+                       :public-key id
+                       :font-file  font-file
+                       :theme      theme}))]
      (profile.utils/photo contact))))
 
 (re-frame/reg-sub

--- a/src/status_im/subs/contact.cljs
+++ b/src/status_im/subs/contact.cljs
@@ -5,10 +5,10 @@
     [legacy.status-im.ui.screens.profile.visibility-status.utils :as visibility-status-utils]
     [quo.theme]
     [re-frame.core :as re-frame]
-    [status-im.common.pixel-ratio :as pixel-ratio]
     [status-im.constants :as constants]
     [status-im.contexts.profile.utils :as profile.utils]
     [status-im.subs.chat.utils :as chat.utils]
+    [status-im.subs.contact.utils :as contact.utils]
     [utils.address :as address]
     [utils.collection]
     [utils.i18n :as i18n]))
@@ -37,50 +37,6 @@
  (fn [multiaccount]
    (get multiaccount :profile-pictures-visibility)))
 
-(defn- replace-contact-image-uri
-  [contact port public-key font-file theme]
-  (let [{:keys [images ens-name customization-color]} contact
-        images
-        (reduce (fn [acc image]
-                  (let [image-name (:type image)
-                        clock      (:clock image)
-                        options    {:port port
-                                    :ratio pixel-ratio/ratio
-                                    :public-key
-                                    public-key
-                                    :image-name
-                                    image-name
-                                    ; We pass the clock so that we reload the
-                                    ; image if the image is updated
-                                    :clock
-                                    clock
-                                    :theme
-                                    theme
-                                    :override-ring?
-                                    (when ens-name false)}]
-                    (assoc-in acc
-                     [(keyword image-name) :config]
-                     {:type    :contact
-                      :options options})))
-                images
-                (vals images))
-
-        images (if (seq images)
-                 images
-                 {:thumbnail
-                  {:config {:type    :initials
-                            :options {:port                port
-                                      :ratio               pixel-ratio/ratio
-                                      :public-key          public-key
-                                      :override-ring?      (when ens-name false)
-                                      :uppercase-ratio     (:uppercase-ratio
-                                                            constants/initials-avatar-font-conf)
-                                      :customization-color customization-color
-                                      :theme               theme
-                                      :font-file           font-file}}}})]
-
-    (assoc contact :images images)))
-
 (defn- enrich-contact
   ([contact] (enrich-contact contact nil nil))
   ([{:keys [public-key] :as contact} setting own-public-key]
@@ -103,7 +59,12 @@
 (defn- reduce-contacts-image-uri
   [contacts port font-file theme]
   (reduce-kv (fn [acc public-key contact]
-               (let [contact (replace-contact-image-uri contact port public-key font-file theme)]
+               (let [contact (contact.utils/replace-contact-image-uri
+                              {:contact    contact
+                               :port       port
+                               :public-key public-key
+                               :font-file  font-file
+                               :theme      theme})]
                  (assoc acc public-key contact)))
              {}
              contacts))
@@ -230,7 +191,12 @@
   [_ contact-identity ens-name port font-file theme]
   (let [contact (enrich-contact
                  (public-key-and-ens-name->new-contact contact-identity ens-name))]
-    (replace-contact-image-uri contact port contact-identity font-file theme)))
+    (contact.utils/replace-contact-image-uri
+     {:contact    contact
+      :port       port
+      :public-key contact-identity
+      :font-file  font-file
+      :theme      theme})))
 
 (re-frame/reg-sub
  :contacts/current-contact
@@ -250,7 +216,10 @@
  :contacts/contact-by-identity
  :<- [:contacts/contacts]
  (fn [contacts [_ contact-identity]]
-   (get contacts contact-identity {:public-key contact-identity})))
+   (get
+    contacts
+    contact-identity
+    (contact.utils/build-contact-from-public-key contact-identity))))
 
 (re-frame/reg-sub
  :contacts/contact-two-names-by-identity
@@ -288,7 +257,7 @@
                           (assoc public-key current-contact))]
     (->> members
          (map #(or (get all-contacts %)
-                   {:public-key %}))
+                   (contact.utils/build-contact-from-public-key %)))
          (sort-by (comp string/lower-case
                         (fn [{:keys [primary-name name alias public-key]}]
                           (or primary-name

--- a/src/status_im/subs/contact/utils.cljs
+++ b/src/status_im/subs/contact/utils.cljs
@@ -1,0 +1,54 @@
+(ns status-im.subs.contact.utils
+  (:require
+    [native-module.core :as native-module]
+    [status-im.common.pixel-ratio :as pixel-ratio]
+    [status-im.constants :as constants]
+    [utils.address :as address]))
+
+(defn replace-contact-image-uri
+  [{:keys [contact port public-key font-file theme]}]
+  (let [{:keys [images ens-name customization-color]} contact
+        images
+        (reduce (fn [acc image]
+                  (let [image-name (:type image)
+                        clock      (:clock image)
+                        options    {:port           port
+                                    :ratio          pixel-ratio/ratio
+                                    :public-key     public-key
+                                    :image-name     image-name
+                                    ; We pass the clock so that we reload the
+                                    ; image if the image is updated
+                                    :clock          clock
+                                    :theme          theme
+                                    :override-ring? (when ens-name false)}]
+                    (assoc-in acc
+                     [(keyword image-name) :config]
+                     {:type    :contact
+                      :options options})))
+                images
+                (vals images))
+
+        images (if (seq images)
+                 images
+                 {:thumbnail
+                  {:config {:type    :initials
+                            :options {:port                port
+                                      :ratio               pixel-ratio/ratio
+                                      :public-key          public-key
+                                      :override-ring?      (when ens-name false)
+                                      :uppercase-ratio     (:uppercase-ratio
+                                                            constants/initials-avatar-font-conf)
+                                      :customization-color customization-color
+                                      :theme               theme
+                                      :font-file           font-file}}}})]
+
+    (assoc contact :images images)))
+
+
+(defn build-contact-from-public-key
+  [public-key]
+  (when public-key
+    (let [compressed-key (native-module/serialize-legacy-key public-key)]
+      {:public-key     public-key
+       :compressed-key compressed-key
+       :primary-name   (address/get-shortened-compressed-key (or compressed-key public-key))})))

--- a/src/tests/test_utils.cljs
+++ b/src/tests/test_utils.cljs
@@ -40,6 +40,8 @@
   (clj->js
    {:sha3
     (fn [s] (.sha3 native-status s))
+    :serializeLegacyKey
+    (fn [s] (.serializeLegacyKey native-status s))
     :setBlankPreviewFlag
     identity
     :encodeTransfer

